doc/sphinx/source/esmvalcore/preprocessor.inc

```diff
@@ -107,32 +290,41 @@ See also :func:`esmvalcore.preprocessor.seasonal_mean`.
 6. annual_mean
 --------------
 
-This function produces an annual or a decadal mean. The only argument is the
-decadal boolean switch. When this switch is set to true, this function
-will output the decadal averages.
+This function produces an annual or a decadal mean. It takes a single boolean
+switch as argument:
+* decadal: calculate decadal averages.
```

Comment:
At the moment this is a little confusing, since the key word "annual" is not shown anymore in the example. Not sure if it would be better to have the example as "* annual: calculares annual averages" to avoid that confusion. But then you might have the confusion on how to call decadal means... Maybe just add both options? Would just be one line more...

Reply:
`annual_mean` calculates both annual (default) and decadal average.

Reply:
But not at the same time, right? And how does it work with the annual mean if it is the default? Do I not have to provide "annual" then?

Reply:
@ledm I guess that's the default.

Reply:
See above.